nmostesting/suites/IS0501Test.py

```diff
@@ -1026,7 +1026,7 @@ def test_42(self, test):
                 url = self.url + "single/senders/{}/transportfile".format(sender)
                 valid, response = self.do_request("GET", url)
                 if valid and response.status_code == 200:
-                    valid, message = check_content_type(response.headers, "application/sdp")
+                    valid, message = check_content_type(response.headers, "application/sdp")
```

Comment:
Not clear in GitHub view what the difference is on this line or others in this file. Please revert all changes to this file.

Reply:
It was an end of line change. We'll make sure to check them in the future.